### ISSUE TITLE
Problem with using two ESP8266 and websockets

### ISSUE THREAD

**Issue by renno-bih** [Links2004/arduinoWebSockets#5]
I use this websocket server on my ESP8266 and I'm trying to connect from my PC in order to read sensor data. Sometimes websocket connection fails but when I click reconnect on my PC it connects again... it is not major problem
But when I connect another ESP8266 at the same network both ESP freezes. Main loop is executing but websocket server freezes. Nothing helps but restart.

Any ideas to solve this problem? What causes freezing when another websocket server is active at the same network?

**Comment by Links2004** [comment]
the freezing make no sense to me, both servers not have any connection to each other.
have both ESP get different IPs on the network?

the dropping of a connection is a browser based problem, the browser has an timeout.
If you not use the connection for some time the browser drop is automatically.
to stop this you need to send some data from time to time.
or simply add a auto reconnect to your JavaScript code to reconnect in case of disconnect , timeout or error (best way I think).

with the examples here: https://github.com/Links2004/arduinoWebSockets/tree/master/tests
i had it running for days without and disconnects.

**Comment by renno-bih** [comment]
Links2004, thanks for fast reply.
As I mentioned, when I get disconnected it is not major problem. I can easily reconnect as a client.

But I'm having other problems. When I turn on more ESP8266 on the same network they all freeze and I cannot reconnect with client anymore.
Looks like I found cause of the problem.
```WiFiUdp``` library was causing a problem. 
As soon as I commented ```Udp.begin(localPort);``` in setup function it started to work properly.

Hmmm, strange.
